.github/ISSUE_TEMPLATE/team_issues.yaml

```diff
@@ -0,0 +1,43 @@
+name: Team issue template
+description: This template is reserved for team members.
+title: "Keep the title concise. Use the common language we can use to identify the issue."
+body:
+  - type: textarea
+    attributes:
+      label: What?
+      description: A concise (or detailed) description of this issue's goal.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Why?
+      description: A concise (or detailed) explanation of why we need this issue.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Suggestion
+      description: Do you have a suggestion to close the issue?
```

Comment:
Can you clarify what is the intention of this `Suggestion` section?

Reply:
Something like this https://github.com/grafana/xk6-browser/issues/1025. See the suggestion section there.

Reply:
Since you have "What" and "Why", you could change this to "How" instead of "Suggestion".

The description could be reworded to something like "Do you have any suggestions/solutions on how to solve the issue?"

Reply:
I liked this one, thx!